tests/integration/test_task_pool.py

```diff
@@ -1393,8 +1393,8 @@ async def test_set_outputs_live(
             'runtime': {
                 'foo': {
                     'outputs': {
-                        'x': 'x',
-                        'y': 'y'
+                        'x': 'xylophone',
```

Comment:
Unfortunately this change reveals a second bug caused by the message and the output having the same value. 

I'm working on getting to the bottom of the why, but it looks to me like it's in the three `satisfy_me` methods.

`prerequisite.satisfied` seems to use a key of `(cycle, name, message)` (I checked on master too), but the tokens being passed to this method are looking for `(cycle, name, output)`.